docs/design.md

```diff
@@ -106,7 +106,7 @@ Netlink is also used to manage the routing table in the namespace.
 
 ## API
 
-Drivers are essentially an extension of libnetwork and provides the actual implementation for all of the LibNetwork APIs defined above. Hence there is an 1-1 correspondence for all the `Network` and `Endpoint` APIs, which includes :
+Drivers is essentially an extension of libnetwork and provides the actual implementation for all of the LibNetwork APIs defined above. Hence there is an 1-1 correspondence for all the `Network` and `Endpoint` APIs, which includes :
```

Comment:
I think this should be "are" (because drivers is plural), but you can change "provides" to "provide" instead